src/JS/store.js

```diff
@@ -23,6 +25,31 @@ class Store {
       localStorage.setItem('taskList', JSON.stringify(tasks));
     });
   }
+
+  static deleteCompTask() {
```

Comment:
- Kindly create a function for deleting a task (remove an element from the array).

Reply:
hi carlos 
i belive in it's working now and remove any element or elements from an array